tests/kani/Loops/loop_with_drop.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! Ensure that Kani correctly unwind the loop with drop instructions which may .
```

Comment:
"unwind" -> "unwinds"

Also, the sentence isn't finished?

Reply:
oops.